src/main/kotlin/net/ccbluex/liquidbounce/features/command/builder/Parameters.kt

```diff
@@ -79,15 +79,11 @@ fun moduleParameter(name: String = "module", validator: (Module) -> Boolean = {
         }
 }
 
-fun playerParameter(name: String = "player"): ParameterBuilder<String> {
+fun playerParameter(name: String = "playerName"): ParameterBuilder<String> {
     return ParameterBuilder
         .begin<String>(name)
         .verifiedBy(ParameterBuilder.STRING_VALIDATOR)
-        .autocompletedWith { _, _ ->
-            mc.networkHandler?.playerList?.map { playerListEntry ->
-                playerListEntry.profile.name
-            } ?: emptyList()
-        }
+        .useMinecraftAutoCompletion()
```

Comment:
This is a thoughtless change that has side effects: due to certain factors you can't auto complete on some servers making it impossible to auto complete names with minecraft.

Reply:
He practically changed `useMinecraftAutoCompletion` to be the same code that is used there.